src/main/java/com/squareup/javawriter/ParameterWriter.java

```diff
@@ -18,15 +18,24 @@
 import java.io.IOException;
 
 public final class ParameterWriter extends VariableWriter {
+  private boolean isVararg;
+
   ParameterWriter(TypeName type, String name) {
     super(type, name);
   }
 
+  public void setIsVararg(boolean isVararg) {
```

Comment:
In the immutable-builder world this could just be `asVarargs()` on the type (not builder) and return a new instance where it was true...